staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -130,6 +132,9 @@ type GenericAPIServer struct {
 	swaggerConfig *swagger.Config
 	openAPIConfig *openapicommon.Config
 
+	// Enables updating OpenAPI spec using update method.
+	OpenAPIService *openapi.OpenAPIService
```

Comment:
The generic API server repo is for supporting the common case for a "normal" API server.  No normal API server will need this update power, so this should not be on the struct here. 

The aggregator can have its own special endpoint to avoid muddying the "easy path" implementation.